.gitignore

```diff
@@ -1,3 +1,6 @@
 *.swp
 nbproject
 pkg
+.rvmrc
+.bundle
+Gemfile.lock
```

Comment:
Why ignoring the Gemfile.lock?

Reply:
It wasn't me. I came from an external commit that I cherry picked.